### PR TITLE
04 fe global layouts

### PR DESCRIPTION
**Pull request by rjeantet**
Integrated react router and consequently changes:
- npm i react-router
- Adding routing to index.js & App.js
- new pages created (home, about & authentication component login)
- moved content of App.js to home.js 

+ quick fix opacity paper component

Note: did not add Provider from redux yet as I want to study redux documentation first sometimes this week (not there yet in my course ^_^!). Will be added next week.

### CODE REVIEW

File: reactApp/src/components/authentication/Login.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function Login() {
+  return (
+    <>
+      <h1>(MODAL) Login with your Naturdoc Account </h1>
```
Comment (somahargitai) [inline reactApp/src/components/authentication/Login.js:4]: For a login Modal you can use a Dialog MUI component

https://mui.com/material-ui/react-dialog/
https://mui.com/material-ui/react-modal/

Comment (somahargitai) [reply reactApp/src/components/authentication/Login.js:4]: You can put the navigation bar and the footer into a Higher-Order Component, so you can easily have these fixed elements for all the pages without repeating them everywhere

**Review by somahargitai — APPROVED** [review]
Amazing progress!

File: reactApp/src/components/authentication/Login.js (hunk 1 above)
Comment (rjeantet) [reply reactApp/src/components/authentication/Login.js:4]: Thank you Soma for the reviews and encouragements! Thank you for the tip, will definitely use higher order components as templates and refactor a bit all this ^_^

**Pull request merged**